CHANGELOG.md

```
@@ -4,6 +4,53 @@ All notable changes to this project will be documented in this file.
+- [ ] Place creation
+- [ ] Tag creation + management
+- [ ] Place privacy zones
+- [ ] Settings panel is scrollable
+- [ ] Family members can enable their location sharing and see each other on the map
+- [ ] Home location
+
+# OIDC and KML support release
+
+To configure your OIDC provider, set the following environment variables:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)